src/main/scala/ztis/model/ModelBuilder.scala

```diff
@@ -1,21 +1,21 @@
 package ztis.model
 
```

Comment:
OK, I see that there are only some formatting changes so it will not impact the functionality.